managed/src/main/java/com/yugabyte/yw/controllers/UniverseController.java

```diff
@@ -1762,4 +1781,43 @@ private UniverseDefinitionTaskParams bindFormDataToTaskParams(ObjectNode formDat
     formNode.remove(listType);
     return gflagMap;
   }
+
+  // TODO(bhavin192): is there any other place for this function?
+  /**
+   * Throw an exception if the given provider has an AZ with
+   * KUBENAMESPACE in the config and the provdier has a cluster
+   * associated with it. Providers with namespace setting don't
+   * support multiple clusters.
+   * @param provider Provider object
+   */
+  private void checkK8sProviderAvailability(Provider provider) {
+    boolean isNamespaceSet = false;
+    for (Region r : Region.getByProvider(provider.uuid)) {
+      for (AvailabilityZone az : AvailabilityZone.getAZsForRegion(r.uuid)) {
+        if (az.getConfig().containsKey("KUBENAMESPACE")) {
+          isNamespaceSet = true;
+        }
+      }
+    }
+
+    if (isNamespaceSet) {
+      for (Universe uu : Universe.getAllUuids()) {
```

Comment:
minor: name these as thisUniv and otherUniv to make it clear instead of uu and u

Reply:
Made it `allUniverseUUIDs` and `u`.